src/js/hyperschedule.js

```diff
@@ -655,13 +720,34 @@ function attachListeners()
     placeholder: createCourseEntity("placeholder").outerHTML,
   });
   printButton.addEventListener("click", downloadPDF);
+  settingsButton.addEventListener("click", showSettingsModal);
+
   selectedCoursesList.addEventListener("sortupdate", readSelectedCoursesList);
   selectedCoursesList.addEventListener("sortstart", () => {
     gCurrentlySorting = true;
   });
   selectedCoursesList.addEventListener("sortstop", () => {
     gCurrentlySorting = false;
   });
+
+  // for (var i = 0; i < conflictCoursesRadios; i++) {
+  //   conflictCoursesRadios[i].addEventListener("click", () => {
+  //     gGreyConflictCourses = gGreyConflictCoursesOptions[i];
+  //   })
+  // }
```

Comment:
What was wrong with this? Did you just forget `.length`?

Reply:
I'm not sure -- I tried the following and it didn't seem to work. I think it's to do with i=3 when the event is triggered.

`for (var i = 0; i < conflictCoursesRadios.length; i++) {
    conflictCoursesRadios[i].addEventListener("click", () => {
      gGreyConflictCourses = greyConflictCoursesOptions[i];
      toggleConflictCourses();
    })
  }`

Reply:
Looks like [an obscure misfeature of JavaScript](https://codeburst.io/asynchronous-code-inside-an-array-loop-c5d704006c99). Try using `let` instead of `var`. In fact, `let` is basically always preferred to `var`, for this and other reasons.

Reply:
This should be resolved in the newest commit